renovate.json

```diff
@@ -1,5 +1,15 @@
 {
   "extends": [
     "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPackageNames": ["org.codehaus.groovy:groovy-dateutil", "org.codehaus.groovy:groovy-all"],
+      "allowedVersions": "<3"
```

Comment:
Both spock and JenkinsPipelineUnit depend on groovy.

JenkinsPipelineUnit seems to track the Jenkins groovy version, forcing it to be upgraded seems risky.
so lets not force it to use above 3 till JenkinsPipelineUnit uses 3.x